cloudplatform/install/rhiot-cloud-platform

```diff
@@ -164,16 +165,29 @@ else
     exe docker start openshift-server
 fi
 
+### OpenShift: authentication. TODO: I'll hard-code the username and password, but we can later change to 
+### receive from parameters.
+exe docker exec -it openshift-server oc login -u admin -p admin https://localhost:8443
+
+### OpenShift: create rhiot project
+exe docker exec -it openshift-server oc new-project rhiot --display-name="Rhiot Cloud Platform" --description="Rhiot Cloud Platform contains all services required in Rhiot Architecture." 
+
 ### MongoDB
 
 log_info "MongoDB"
 if [ $(docker ps -a | grep mongodb_data | wc -l) -eq 0 ]; then
     log_info "MongoDB data volume doesn't exist. Creating..."
-    exe docker run -v /data/db --name mongodb_data busybox true
+    #exe docker run -v /data/db --name mongodb_data busybox true
+    exe docker exec -it openshift-server oc new-app mongodb-persistent--param=DATABASE_SERVICE_NAME=rhiot --param=MONGODB_USER=rhiot --param=MONGODB_PASSWORD=rhiot
```

Comment:
Maybe your openshift instance doesn't have the mongodb-persistent template. Check if you have with the comand `docker -ti openshift-server oc new-app search mongo | grep mongodb-persistent`